test-e2e/members.js

```diff
@@ -25,8 +25,7 @@ test('getting yourself after creating project', async (t) => {
   const deviceInfo = manager.getDeviceInfo()
   const project = await manager.getProject(await manager.createProject())
 
-  const me = await project.$member.getById(project.deviceId)
-
+  const { joinedAt: _, ...me } = await project.$member.getById(project.deviceId)
```

Comment:
Can we test the value of `joinedAt` somehow?

Reply:
yeah, was thinking about this, see -> https://github.com/digidem/mapeo-core-next/pull/745#discussion_r1715519655